.travis.yml

```diff
@@ -2,9 +2,10 @@ sudo: false
 
 language: go
 
+go_import_path: github.com/prometheus/graphite_exporter
```

Comment:
This should be below `go:` like we have it in other `.travis.yml` files.

Reply:
Done I hope ;)